### PR TITLE
bug fix - missing rule in 2.03 activity and org schemas

### PR DESCRIPTION
**Pull request by amy-silcock**
At 2.03 the wording for the default language xml:lang attribute was updated. Whist doing this, the rule that the default language must be specified, or all narrative elements must have a language set was missed out.

https://discuss.iatistandard.org/t/language-recommend-use-of-iso-639-1-included-2-03/842

This PR fixes this bug.

### CODE REVIEW

**Review by Ocre42 — APPROVED** [review]
Looks good to me, waiting for review from @PetyaKangalova before merging

**Comment by andylolz** [comment]
Big big +1 for linking to the discuss post in the PR. This is GREAT because it makes it possible to trace a code change to the discussion that triggered it.

Thanks Amy!

Just to add one more link in case it provides some further explanation: the change Amy describes was made in #401. The diff shows that the text in question was missed out.

**Review by PetyaKangalova — APPROVED** [review]
Looks good to me and fixes the bug @Ocre42 . Please go ahead and merge.

**Pull request merged**
